pkgs/desktops/gnome/extensions/extensionRenames.nix

```diff
@@ -12,6 +12,9 @@
   "workspace-indicator@gnome-shell-extensions.gcampax.github.com" = "workspace-indicator";
   "horizontal-workspace-indicator@tty2.io" = "workspace-indicator-2";
 
+  "floating-panel@aylur" = "floating-panel";
+  "floating-panel-usedbymyself@wpism" = "floating-panel-2";
```

Comment:
I did not manage to find any source repository for the second one (its EGO still links the original). It generally looks rather fishy, so I suggest deleting it setting it to `null` instead. Maybe with a small comment